packages/flutter_tools/test/general.shard/ios/ios_deploy_test.dart

```diff
@@ -75,8 +75,9 @@ void main () {
         interfaceType: IOSDeviceConnectionInterface.network,
       );
 
+      expect(iosDeployDebugger.logLines, emits('Did finish launching.'));
```

Comment:
I swapped this test file to use the `emits` matcher instead of `Stream.toList()`